presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -3393,6 +3393,23 @@ public void testLogicalExplain()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan(query, LOGICAL));
     }
 
+    @Test
+    public void testJoinDistribution()
+    {
+        /*
+            This test makes sure that the query plan includes the distribution info for joins.
+            The webui live plan displays the distribution type for joins, and the plan details
+            need to be structured like 'Distribution: <type>' for this feature to work.
+        */
+        String query = "SELECT c.custkey\n" +
+                       "FROM customer c JOIN nation n\n" +
+                       "  ON n.nationkey = c.nationkey\n" +
+                       "WHERE n.name = 'UNITED STATES'\n" +
+                       "LIMIT 5";
+        MaterializedResult result = computeActual("EXPLAIN (FORMAT TEXT) " + query);
```

Comment:
avoid `\n` as they don't slightly reduce readability, use spaces instead
also, you can make the lines a bit longer and remove unnecessary WHERE, LIMIT
then you can inline the variable:

```suggestion
        MaterializedResult result = computeActual("EXPLAIN (FORMAT TEXT) SELECT c.custkey FROM customer c JOIN nation n ON n.nationkey = c.nationkey");
```